src/main/java/com/linkedin/kmf/services/JettyService.java

```diff
@@ -9,9 +9,17 @@
  */
```

Comment:
If this is an inactive `Pull Request`, may I have your permission to close it or update?
 😄 
Thanks!

Reply:
@andlaz

Reply:
feel free to update or close @andrewchoi5